### PR TITLE
Updated CC docs - part 1 

### PR DESCRIPTION
**Pull request by Mariangela**
## Purpose
- fixes https://github.com/wso2/docs-apim/issues/5030
- Following fixes for https://github.com/wso2/docs-apim/issues/5014
   - Reviewed and updated en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview.md
   - Reviewed and updated en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/supported-features.md
   - Fixed formatting issues

### CODE REVIEW

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview.md [hunk 1]
```diff
@@ -1,43 +1,51 @@
...
 ## Choreo Connect Components
 
 Choreo Connect consists of the following three components.
 
-### 1. Router
+- [Router](#router)
+- [Enforcer](#enforcer)
+- [Adapter](#adapter)
 
+<<<<<<< HEAD
```
Comment (pubudu538) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview.md:17]: Looks like merge conflcts.

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/http-2.0.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 # HTTP 2.0
```
Comment (pubudu538) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/http-2.0.md:1]: We need to remove this doc. THis is coming from mgw v320 and not applicable for CC v1.

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway.md [hunk 3]
```diff
@@ -33,11 +33,11 @@ In Choreo Connect, version 3.0.x series the open API definition required 2 manda
 
 `x-wso2-basePath` is used to group all HTTP resources of the OpenAPI under a single context. Therefore, it will be exposed as a sub resource of a single API using Choreo Connect. This is how Choreo Connect logically separates one API from the other. The `x-wso2-production-endpoints` refers to the actual backend service implementation of the API. Therefore, the latter mentioned details are mandatory to expose an API using Choreo Connect.
 
-**But with 3.1.0 onwards none of the WSO2 specific open API extensions are mandatory.** Choreo Connect resolves base path and back end URL as below
+**But with 3.1.0 onwards none of the WSO2 specific open API extensions are mandatory.** Choreo Connect resolves base path and back end URL as follows:
```
Comment (pubudu538) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway.md:36]: Let's remove the below line. It is used in v310 Ballerina mgw.

But with 3.1.0 onwards none of the WSO2 specific open API extensions are mandatory

Comment (Mariangela) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway.md]: fixed in https://github.com/wso2/docs-apim/pull/5032/commits/f02f25aa5ccc060afcd096f91bb1273a3fc042a8

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview.md (hunk 1 above)
Comment (Mariangela) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview.md]: Fixed in https://github.com/wso2/docs-apim/pull/5032/commits/f02f25aa5ccc060afcd096f91bb1273a3fc042a8

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/http-2.0.md (hunk 2 above)
Comment (Mariangela) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/http-2.0.md]: fixed in https://github.com/wso2/docs-apim/pull/5032/commits/2b2ceaa1a15bcf07a89001a6684a9824c5eb4a60

**Comment by Mariangela** [comment]
@pubudu538 , I updated this PR based on the comments.

**Review by pubudu538 — APPROVED** [review]

**Pull request merged**